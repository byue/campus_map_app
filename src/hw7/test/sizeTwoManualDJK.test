# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 2 graph.

CreateGraph g

AddNode g a
AddNode g b

FindPath g a b