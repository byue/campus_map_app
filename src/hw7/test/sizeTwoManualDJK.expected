# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 2 graph.

created graph g

added node a to g
added node b to g

path from a to b:
no path found